liquidity-rewards/test/UniswapPoolRewardsTest.js

```diff
@@ -7,6 +7,7 @@ const UniswapPoolRewards = artifacts.require('UniswapPoolRewards');
 
 const _1e18 = new BN('10').pow(new BN('18'));
 const ONE_MONTH = new BN('2629800'); // 1 month = 2629800 seconds
+const ONE_DAY = new BN('86400'); // 1 day = 86400 seconds
 const _1_000 = '10000';
```

Comment:
Should this be 1,000 instead of 10,000?
```suggestion
const _1_000 = '1000';
```

Reply:
Yeah, Thanks for correcting my mistakes!